facebook-commerce.php

```diff
@@ -3753,129 +3546,6 @@ function ajax_display_test_result() {
 		wp_die();
 	}
 
-
-	/**
-	 * Schedules a recurring event to sync products.
-	 *
-	 * @deprecated 1.10.0
-	 */
-	function ajax_schedule_force_resync() {
-
-		wc_deprecated_function( __METHOD__, '1.10.0' );
-		die;
-	}
-
-
-	/**
-	 * Adds an recurring action to sync products.
-	 *
-	 * The action is scheduled using a cron schedule instead of a recurring interval (see https://en.wikipedia.org/wiki/Cron#Overview).
-	 * A cron schedule should allow for the action to run roughly at the same time every day regardless of the duration of the task.
-	 *
-	 * @since 1.10.0
-	 *
-	 * @param int $offset number of seconds since the beginning of the daay
-	 */
-	public function schedule_resync( $offset ) {
-
-		try {
-
-			$current_time         = new DateTime( 'now', new DateTimeZone( wc_timezone_string() ) );
-			$first_scheduled_time = new DateTime( "today +{$offset} seconds", new DateTimeZone( wc_timezone_string() ) );
-			$next_scheduled_time  = new DateTime( "today +1 day {$offset} seconds", new DateTimeZone( wc_timezone_string() ) );
-
-		} catch ( \Exception $e ) {
-			// TODO: log an error indicating that it was not possible to schedule a recurring action to sync products {WV 2020-01-28}
-			return;
-		}
-
-		// unschedule previously scheduled resync actions
-		$this->unschedule_resync();
-
-		$timestamp = $first_scheduled_time >= $current_time ? $first_scheduled_time->getTimestamp() : $next_scheduled_time->getTimestamp();
-
-		// TODO: replace 'facebook-for-woocommerce' with the plugin ID once we stat using the Framework {WV 2020-01-30}
-		as_schedule_single_action( $timestamp, self::ACTION_HOOK_SCHEDULED_RESYNC, array(), 'facebook-for-woocommerce' );
-	}
-
-
-	/**
-	 * Removes the recurring action that syncs products.
-	 *
-	 * @since 1.10.0
-	 */
-	private function unschedule_resync() {
-
-		// TODO: replace 'facebook-for-woocommerce' with the plugin ID once we stat using the Framework {WV 2020-01-30}
-		as_unschedule_all_actions( self::ACTION_HOOK_SCHEDULED_RESYNC, array(), 'facebook-for-woocommerce' );
-	}
-
-
-	/**
-	 * Determines whether a recurring action to sync products is scheduled and not running.
-	 *
-	 * @see \as_next_scheduled_action()
-	 *
-	 * @since 1.10.0
-	 *
-	 * @return bool
-	 */
-	public function is_resync_scheduled() {
```

Comment:
Also being used in https://github.com/woocommerce/facebook-for-woocommerce/blob/bf97a40040324359069f4d0190d932e495987923/includes/Lifecycle.php#L191-L193

Reply:
Removed. We are doing things differently now so Lifecycle updates need change.